io/serialport/locales/de/25-serial.json

```diff
@@ -24,7 +24,7 @@
             "startor": ", dann"
         },
         "placeholder": {
-            "serialport": "z. B. /dev/ttyUSB0/"
+            "serialport": "z.B. /dev/ttyUSB0/"
```

Comment:
You're right and I changed it globally, but I use a no-break space (&nbsp;) in HTML helps.
And I keep "z.B." without space in a few config windows because of too less space, where the german "z. B." (with space) as translation of the englisch "e.g." (without space) would no more fit.
Here an interesting article: https://www.typolexikon.de/zum-beispiel/

Reply:
P.S. I just found the same with "d. h."